### PR TITLE
Partial (1/2) spanish translation ch08-03-hash-maps.md

### PR DESCRIPTION
**Pull request by jossue33**
Number of words: 1011
Line: 1-171

### CODE REVIEW

File: second-edition/src/ch08-03-hash-maps.md [hunk 1]
```diff
@@ -144,31 +144,31 @@ for (key, value) in &scores {
...
 
-Although the number of keys and values is growable, each key can only have one
-value associated with it at a time. When we want to change the data in a hash
-map, we have to decide how to handle the case when a key already has a value
-assigned. We could replace the old value with the new value, completely
-disregarding the old value. We could keep the old value and ignore the new
-value, and only add the new value if the key *doesn’t* already have a value. Or
-we could combine the old value and the new value. Let’s look at how to do each
-of these!
+Aunque la cantidad de claves y valores es creíble, cada clave solo puede tener un
```
Comment (thecodix) [inline second-edition/src/ch08-03-hash-maps.md]: 'es creible' should be changed to 'puede crecer'

**Comment by thecodix** [comment]
Hello @jossue33 , do you mind changing the term 'mapa hash' for 'hash map'? No matter if it's in english, it will be better to maintain it that way

**Comment by jossue33** [comment]
@thecodix Todo listo

**Comment by thecodix** [comment]
machine translated

**Pull request merged**